### PR TITLE
Announce 3.0.CR1 pre-release

### PR DESCRIPTION
**Pull request by tkrille**

### CODE REVIEW

File: _posts/2016-06-09-pre-release-osiam-3.0.CR1.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title:  "Pre-Release: OSIAM 3.0.CR1"
+---
+
+Unfortunately, we missed the deadline for OSIAM 3.0.
+We've got stuck on some key changes, but we finally finished our new build and release process today.
+To make up for the delay, we have [tested the release process](https://circleci.com/gh/osiam/osiam/478)
+with an official pre-release of OSIAM: 3.0.CR1.
+This is for all of you, who patiently waited for so long now.
+You can download it from our new [Bintray repository](https://bintray.com/osiam):
```
Comment (dacrome) [inline _posts/2016-06-09-pre-release-osiam-3.0.CR1.md]: Isn't it `https://bintray.com/osiam/downloads/osiam`?

Comment (tkrille) [inline _posts/2016-06-09-pre-release-osiam-3.0.CR1.md]: There are many URLs :) You like the anemic download page more?

Comment (dacrome) [inline _posts/2016-06-09-pre-release-osiam-3.0.CR1.md]: Yes. It links to the osiam download repository and shows the available versions

Comment (tkrille) [inline _posts/2016-06-09-pre-release-osiam-3.0.CR1.md]: Alright.

**Comment by tkrille** [comment]
Updated

**Comment by dacrome** [comment]
Awesome! Thank you!

**Pull request merged**